Common/TableProducer/Converters/trackQA002Converter.cxx

```diff
@@ -0,0 +1,97 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+#include <limits>
+
+#include "Framework/runDataProcessing.h"
+#include "Framework/AnalysisTask.h"
+#include "Framework/AnalysisDataModel.h"
+
+using namespace o2;
+using namespace o2::framework;
+
+struct TrackQAConverter002 {
+  Produces<aod::TracksQA_002> tracksQA_002;
+
+  void process000(aod::TracksQA_000 const& tracksQA_002)
+  {
+    for (const auto& trackQA : tracksQA_000) {
+      tracksQA_002(
```

Comment:
Sorry, but this is clearly a bug which does not compile.

Reply:
Thank you for checking.  I got the modification from Felix, and I assume that I compiled it.
Looks like it was not synced to my build server properly